src/constants/flowdesigner.proptypes.ts

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
```

Comment:
Do we still want to use prop-types in this TS implementation ?

Reply:
This whole file is no longer use in the project itself, but is exposed and used by host app...